LICENSE

```diff
@@ -1,148 +1,188 @@
+  ENTIRE RISK AS TO THE RESULTS AND PERFORMANCE OF THE SOFTWARE AND/OR
+  DERIVATIVE WORKS.  YOU AGREE THAT ILLINOIS SHALL NOT BE HELD LIABLE
+  FOR ANY DIRECT, INDIRECT, CONSEQUENTIAL, OR INCIDENTAL DAMAGES WITH
+  RESPECT TO ANY CLAIM BY YOU OR ANY THIRD PARTY ON ACCOUNT OF OR
+  ARISING FROM THIS LICENSE OR USE OF THE SOFTWARE AND/OR DERIVATIVE
+  WORKS.  YOU SHALL INDEMNIFY AND HOLD HARMLESS ILLINOIS (INCLUDING
+  ITS TRUSTEES, FELLOWS, OFFICERS, EMPLOYEES, STUDENTS, AND AGENTS)
+  AGAINST ANY AND ALL CLAIMS, LOSSES, DAMAGES, AND/OR LIABILITY, AS
+  WELL AS COSTS AND EXPENSES (INCLUDING LEGAL EXPENSES AND REASONABLE
+  ATTORNEYS' FEES) ARISING OUT OF OR RELATED TO YOUR USE, OR INABILITY
+  TO USE, THE SOFTWARE OR SOFTWARE DERIVATIVE WORKS, REGARDLESS OF
+  THEORY OF LIABILITY, WHETHER FOR BREACH OR IN TORT (INCLUDING
+  NEGLIGENCE).
+
+10 Termination.
+
+   10.1 This License is effective until terminated, as provided
+        herein, or until all intellectual property rights in the
+        Software expire.
+
+   10.2 You may terminate this License at any time by destroying all
+        copies of the Software and its derivative works.
+
+   10.3 This License, and the rights granted hereunder, will terminate
+        automatically, and without any further notice from or action
+        by Illinois, if You fail to comply with any obligation of this
+        License.
+
+   10.4 Upon termination, You must immediately cease use of and destroy
+     	all copies of the Software and its derivative works and verify
+     	such destruction in writing.
+
+   10.5 The provisions set forth in Sections 5, 9, 11, 12, 13, 14, 15
+        shall survive termination or expiration of this License.
+
+11 Governing Law. By using this Software, You agree to abide by the
+   laws of the State of Illinois and all other applicable laws of the
+   U.S, including, but not limited to, export control laws, copyright
+   law and the terms of this License.
+
+12 Severability.  If any provision of this License is held to be
+   invalid or unenforceable by a court of competent jurisdiction, such
+   invalidity or unenforceability shall not in any way affect the
+   validity or enforceability of the remaining provisions.
+
+13 Assignment.  You may not assign or otherwise transfer any of Your
+   rights or obligations under this License, without the prior written
+   consent of Illinois.
+
+14 Entire License.  This License represents the parties' entire
+   agreement relating to the Software.  Except as otherwise provided
+   herein, no modification of this License is binding unless in
+   writing and signed by an authorized representative of each party.
+
+15 Waiver.  The failure of either party to enforce any provision of
+   this License shall not constitute a waiver of that right or future
+   enforcement of that or any other provision.
+
+Approved for legal form Illinois Counsel SJA 10/18
+
+University Technology No. TF08023 Commercial Non-Exclusive Software License
+
+
```

Comment:
Stray blank lines.

Reply:
Export from Word creates very strange issues. Should be addressed in the fixup patch.